changelog/Microsoft.Teams.GraphSvc.json

```
@@ -447,6 +447,40 @@
       "CreatedDateTime": "2022-05-24T18:54:16.6879906Z",
       "WorkloadArea": "Teamwork",
       "SubArea": ""
+    },
+    {
+      "ChangeList": [
+        {
+          "Id": "e04528be-94f6-4bc5-9485-9b71da7f4804",
+          "ApiChange": "Resource",
+          "ChangedApiName": "teamsAppSettings",
+          "ChangeType": "Addition",
+          "Description": "Added the [teamsAppSettings](https://docs.microsoft.com/en-us/graph/api/resources/teamsAppSettings?view=graph-rest-beta) resource",
+          "Target": "teamsAppSettings"
+        },
+        {
+          "Id": "e04528be-94f6-4bc5-9485-9b71da7f4804",
+          "ApiChange": "Property",
+          "ChangedApiName": "isChatResourceSpecificConsentEnabled",
+          "ChangeType": "Addition",
+          "Description": "Added the **isChatResourceSpecificConsentEnabled** property to the [teamsAppSettings](https://docs.microsoft.com/en-us/graph/api/resources/teamsAppSettings?view=graph-rest-beta) resource.",
+          "Target": "teamsAppSettings"
+        },
+        {
+          "Id": "e04528be-94f6-4bc5-9485-9b71da7f4804",
+          "ApiChange": "Resource",
+          "ChangedApiName": "teamsAppSettings",
+          "ChangeType": "Addition",
+          "Description": "Added the [teamsAppSettings](https://docs.microsoft.com/en-us/graph/api/resources/teamwork?view=graph-rest-beta) resource",
```

The description should be 
```suggestion
          "Description": "Added the **teamsAppSettings** navigation property to the [teamsAppSettings](https://docs.microsoft.com/en-us/graph/api/resources/teamwork?view=graph-rest-beta) resource",
```

---

"to the teamwork resource" right?